insights/combiners/rsyslog_confs.py

```diff
@@ -0,0 +1,47 @@
+"""
+RsyslogConfAll - files ``/etc/rsyslog.conf`` and ``/etc/rsyslog.d/*.conf``
+==========================================================================
+
+Combiner for accessing all the rsyslog comfiguration files. There may be
+multiple rsyslog configuration, and the main configuration file is
+``/etc/rsyslog.conf``. This combiner will not check same option in multi
+files, user needs to check this situation in plugin if it is necessary.
+
+"""
+from insights.core.plugins import combiner
+from insights.parsers.rsyslog_conf import RsyslogConf
+from .. import LegacyItemAccess
+
+
+@combiner(RsyslogConf)
+class RsyslogAllConf(LegacyItemAccess):
```

Comment:
@wushiqinlou since you are essentially creating a dictionary in `self.data` you should just base this class on `dict` which would provide all of the functions in `LegacyItemAccess`.

Reply:
Updated, please have a check